Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -73,6 +73,11 @@ public float groundedMoveSpeed
 	[SerializeField] float rollCooldownTime = 1f;
 	float rollCooldownTimer = 0f;
 
+	public bool isGrabbing
+	{
+		get{ return WadeUtils.ValidAxisInput("Grab"); }
+	}
+
 	[SerializeField] float slideTurnSpeed = 7f;
 	public Vector3 rigVelocity; // used to visualize velocity in inspector
```

Comment:
If rigVelocity is just for visualizing in the inspector, it should be private and serialized, not public.